tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/Net/ASM/Parser/GraphQLNameProxy.cs

```diff
@@ -0,0 +1,19 @@
+// <copyright file="GraphQLNameProxy.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.Net.ASM;
```

Comment:
Two minor remarks:
- Looks like the namespace doesn't match the folder structure
- For HotChocolate these classes are called AST, here they're called Parser, is there a reason for this inconsistency?

Reply:
1) Oops changed the files folder at the end

2) In graphql-dotnet, the library that do the parsing and the AST is called `parser` and in HotChocolate, there was no particular name but there is only AST nodes inside that folder. I can make the folders name consistant.